src/VisualStudio/Setup.Next/source.extension.vsixmanifest

```diff
@@ -11,6 +11,11 @@
     <InstallationTarget Version="[15.0,]" Id="Microsoft.VisualStudio.VWDExpress" />
     <InstallationTarget Version="[15.0,]" Id="Microsoft.VisualStudio.VSWinExpress" />
   </Installation>
+  <Installer>
+    <Actions>
+      <Action Type="Ngen" Path="Microsoft.CodeAnalysis.EditorFeatures.Next.dll" />
+    </Actions>
+  </Installer>
   <Dependencies>
```

Comment:
I'll add NGEN for Telemetry.

Regarding Setup.Next, this package doesn't contain any assemblies, only pkgdef/redirects so there's nothing to NGEN so the mangled path is fine.